roles/os_hardening/tasks/minimize_access.yml

```diff
@@ -73,12 +73,23 @@
     mode: "0750"
   when: '"change_user" not in os_security_users_allow'
 
+# we have to define this combined variable here, because when defining it
+# in the defaults like this:
+# proc_mnt_options: rw,nosuid,nodev,noexec,relatime,hidepid={{ hidepid_option }}
+# the role argument validator complains about the missing hidepid_option-variable.
+# If we define the hidepid_option-variable in the defaults, we cannot
+# overwrite it anymore in the vars (for some reason).
+
+- name: Define proc_mnt_options including hidepid_option
+  ansible.builtin.set_fact:
+    _proc_mnt_options: "{{ proc_mnt_options }},hidepid={{ hidepid_option }}"
```

Comment:
this might generate problems. If someone overwrote `proc_mnt_options` to not include `hidepid` anymore, this would reintroduce it. Possibly breaking change.

Reply:
Yes, this will be a breaking change. Setting it to `0` effectively disables it.

Reply:
hmm, now the `hidepid_option` is missing in docs.

Reply:
Yes, because it's not a `default` anymore but a `var`. Only `defaults` are described in the argument spec and thus in the readme.